src/nni_manager/training_service/kubernetes/kubeflow/kubeflowApiClient.ts

```diff
@@ -123,5 +146,24 @@ class PytorchOperatorClientV1Beta1 extends KubernetesCRDClient {
     }
 }
 
+class PytorchOperatorClientV1Beta2 extends KubernetesCRDClient {
```

Comment:
Use either 'PyTorch' or 'pytorch',  name 'Pytorch' is barely seen.

Reply:
changed to PyTorch